src/misc/00_intro.rst

```diff
@@ -24,6 +24,8 @@ Glossary
   CRAN
     **C**\ omprehensive **R** **A**\ rchive **N**\ etwork. CRAN is a network of FTP and web servers around the world that store identical, up-to-date, versions of code and documentation for R. `Learn More <https://cran.r-project.org/>`_.
 
+  CFEP  
+    **C**\ onda **F**\ orge **E**\ nhancement **P**\ roposal. This CFEP contains the index of all conda-forge Enhancement Proposals, known as CFEPs. All major changes to the project should be submitted as CFEP. 
```

Comment:
```suggestion
    **C**\ onda **F**\ orge **E**\ nhancement **P**\ roposal. As conda-forge grows, all major changes to the project from a technical standpoint as well as to address social topics such as governance and expected conduct should be submitted as a CFEP. `Learn More <https://conda-forge.org/docs/orga/cfep-index.html/>`_.
```

Reply:
I  tried to make that short,😅 my bad its better to add the same as mentioned in  https://github.com/conda-forge/cfep
Thanks @viniciusdc

Reply:
due to its contents and how it changes conda-forge, the acceptance of a CFEP depends on a voting session from core.

Reply:
> I tried to make that short, my bad its better to add the same as mentioned in https://github.com/conda-forge/cfep
> Thanks @viniciusdc

no worries it is good to be brief too, I just find this line "This CFEP contains the index of all conda-forge Enhancement Proposals" quite strange from the rest of the text, as it seans you are refering to an specifq CFEP (or an index containing then, and that's not what a CFEP means)

Reply:
ahhh actually I took the reference from https://www.python.org/dev/peps/  as this too works in the same way. But that's true it's not conveying the purpose